build-aux/m4/ax_boost_system.m4

```diff
@@ -109,7 +108,7 @@ AC_DEFUN([AX_BOOST_SYSTEM],
 
             fi
             if test "x$ax_lib" = "x"; then
-                AC_MSG_ERROR(Could not find a version of the boost_system library!)
+                AC_MSG_ERROR(Could not find a version of the library!)
```

Comment:
This reverts (part of) 25333a26197b55c23debb38740ed875999aa795a but I don't know if we care.

Reply:
I didn't realize we were using a patch on top of https://www.gnu.org/software/autoconf-archive/ax_boost_base.html

Reply:
Oops: #17010